src/constants.js

```diff
@@ -17,7 +17,7 @@ export function generatePrefixes(options) {
     truckmap: 'truckmap://',
     waze: 'waze://',
     yandex: 'yandexnavi://',
-    moovit: 'moovit://',
+    moovit: isIOS ? 'moovit://' : 'moovit://directions',
```

Comment:
I think we can include the hostname for both platforms, iOS will still only check the URL scheme.

Reply:
Made tests and you are right, we can unify it between platforms. Fixed.